librz/arch/isa/x86/common.c

```diff
@@ -884,7 +887,7 @@ RZ_IPI RzILOpBool *x86_il_is_sub_underflow(RZ_OWN RZ_NONNULL RzILOpPure *res, RZ
 	// res & !x & y
 	RzILOpBool *rnxy = AND(AND(DUP(resmsb), INV(DUP(xmsb))), DUP(ymsb));
 	// or = nrxny | rnxy
-	RzILOpBool * or = OR(nrxny, rnxy);
+	RzILOpBool *or = OR(nrxny, rnxy);
```

Comment:
This looks like you maybe use another `clang-format` version. Currently the CI uses `clang-format-16`